imagecraft/pack/diskutil.py

```diff
@@ -254,5 +255,6 @@ def inject_partition_into_image(
         f"of={str(imagepath)}",
         f"bs={disk_size.sector_size}",
         f"seek={sector_offset}",
+        "status=none",
```

Comment:
Should the `run` function make an `emit.trace` call about what it's running? Just thinking that this way we'll get better insight if one of these commands takes a long time.

Reply:
The `run` function was previously using emit but when used as the callable for the chroot it was hanging (at verbosity levels above `debug`) (see https://github.com/canonical/imagecraft/pull/127/commits/51a9d979f0e97d996d766f18dab0deb6a9a14e1f#diff-c4c127f022fd38e4a8535ac927b9d4809e747ee1943fae4a13bb9203ab663bd7L44). At the time I wanted to fix it because it was basically breaking imagecraft if not used bit the proper verbosity level but we can revisit that. 

For now I am keeping calls to emit out of this function.

I reworked the call to `dd`, let me know what you think.

Reply:
Thanks for the explanation. I'm fine with that!